src/audio/copier/copier_host.c

```diff
@@ -100,6 +101,9 @@ int copier_host_create(struct comp_dev *parent_dev, struct copier_data *cd,
 		ret = init_pipeline_reg(parent_dev);
 		if (ret)
 			goto e_conv;
+
+		/* set max sink count for playback */
+		mod->max_sinks = IPC4_COPIER_MODULE_OUTPUT_PINS_COUNT;
```

Comment:
max_sinks need set for different cases, it depend on direction, can we move to module_adapter and set it together?

Reply:
Cannot be moved. This is module specific